src/commands/waw/workbench/open.ts

```diff
@@ -74,7 +74,8 @@ export default class ConnectedAppCreate extends SfdxCommand {
 
     const accessToken = conn.accessToken;
     const instanceUrl = conn.instanceUrl;
-    const serverUrl = `${instanceUrl}/services/Soap/u/46.0`;
+    const apiVersion = conn.version;
+    const serverUrl = `${instanceUrl}/services/Soap/u/${apiVersion}`;
```

Comment:
@amphro  looks like it actually may be conn.version according to the jsforce docs http://jsforce.github.io/jsforce/doc/Connection.html

Reply:
sfdx-core adds on that method, but that works too. https://forcedotcom.github.io/sfdx-core/classes/connection.html#getapiversion